src/Molecules/UserSelect/UserSelect.test.jsx

```diff
@@ -0,0 +1,166 @@
+  setStart:                () => {},
+  setEnd:                  () => {},
+  commonAncestorContainer: {
+    nodeName:      'BODY',
+    ownerDocument: document,
+  },
+})
+
+const StateHolder = ({ children = () => {}, state = [] }) => {
+  const [values, onChange] = useState(state)
+  const [errors, setErrors] = useState([])
+  const onError = data => setErrors(errors.concat(data))
+
+  return <>
+    <div data-testid="values">
+      {values.map(value => <div key={value} data-testid={`value-${value}`} />)}
+    </div>
+    <div data-testid="errors">
+      {errors.map(({ error }) => <div key={error} data-testid={`error-${error}`} />)}
+    </div>
+    {children({ values, onChange, onError })}
+  </>
+}
+
+it('should show label when rendered', () => {
+  const { getByText } = render(
+    <UserSelect label="Hello world" />,
+  )
+
+  expect(getByText('Hello world')).toBeVisible()
+})
+
+it.each([
+  '//material-ui.com/static/images/avatar/1.jpg',
+  'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7',
+])('should display avatar when provided', (avatar, id = 1) => {
+  const { getByAltText } = render(
+    <UserSelect values={[id]}>
+      <User value={id} avatar={avatar} />
+    </UserSelect>,
+  )
+
+  expect(getByAltText(avatar)).toBeVisible()
+})
+
+it.each([
+  'success',
+  'warning',
+])('should display correct color for a given status when provided', (status, id = 1) => {
+  const { getByText } = render(
+    <UserSelect values={[id]}>
+      <User value={id} status={status}>{status}</User>
+    </UserSelect>,
+  )
+
+  expect(getByText(status).parentNode).toHaveAttribute('class', expect.stringContaining(`-${status}-`))
+})
+
+it('should add item in the list', async () => {
+  const { getByRole, getByTestId, findByText, findByTestId } = render(
+    <StateHolder>
+      {({ ...props }) => (
+        <UserSelect {...props}>
+          <User value={1}>user</User>
+        </UserSelect>
+      )}
+    </StateHolder>,
+  )
+
+  user.click(getByRole('combobox'))
+  user.click(await findByText('user'))
+
+  expect(getByTestId('values')).toContainElement(await findByTestId('value-1'))
+})
+
+it('should remove item of the list', async () => {
+  const { getByRole, getByTestId, findAllByText } = render(
+    <StateHolder state={[1]}>
+      {({ ...props }) => (
+        <UserSelect {...props}>
+          <User value={1}>user</User>
+        </UserSelect>
+      )}
+    </StateHolder>,
+  )
+
+  user.click(getByRole('combobox'))
+  user.click((await findAllByText('user')).find(node => !node.classList.contains('MuiChip-label')))
+
+  expect(getByTestId('values')).toBeEmpty()
+})
+
+it('should remove item of the list using chip', () => {
+  const { getByText, getByTestId } = render(
+    <StateHolder state={[1]}>
+      {({ ...props }) => (
+        <UserSelect {...props}>
+          <User value={1}>user</User>
+        </UserSelect>
+      )}
+    </StateHolder>,
+  )
+
+  user.click(getByText('user').nextSibling)
```

Comment:
This will break when we'll change the close action position. But I think it's ok for now